src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -165,8 +165,6 @@ public static void TestDispose()
         }
 
         [Fact]
-        // Crashing on macOS 10.13 Beta
-        [ActiveIssue(21436, TestPlatforms.OSX)]
```

Comment:
This test will ::fingers_crossed:: fail tonight.  But with an xunit failure, not a segfault.  Then it and its bretheren will get a new issue number created and we'll disable them with a 10.13-specific PlatformDetection test.